packages/suite/src/components/suite/modals/ReviewTransaction/components/getIsUpdatedSendFlow.ts

```diff
@@ -0,0 +1,21 @@
+import { TrezorDevice } from '@suite-types/index';
+import { DeviceModel, getDeviceModel } from '@trezor/device-utils';
+import { versionUtils } from '@trezor/utils';
+
+export const getIsUpdatedSendFlow = (device: TrezorDevice) => {
+    const { firmwareRelease } = device;
+    if (!firmwareRelease) {
+        return false;
+    }
+
+    const model = getDeviceModel(device);
+
+    const {
+        release: { version },
+    } = firmwareRelease;
+
+    const isWithUpdatedFlow =
+        model === DeviceModel.T1 && versionUtils.isNewerOrEqual(version, '2.6.0');
```

Comment:
DeviceModel.T1 is newer newer 2
```suggestion
        model === DeviceModel.TT && versionUtils.isNewerOrEqual(version, '2.6.0');
```

Reply:
7b165ac6d